src/lib.rs

```diff
@@ -5,9 +5,8 @@
 #![allow(non_upper_case_globals)]
 
 pub mod ast;
+pub mod codegen;
 pub mod fns;
-pub mod jit;
+pub mod ir;
+pub mod quoting;
 pub mod run;
-

```

Comment:
I dont remember removing this code?

Reply:
This module was actually removed at some point but because this is feature gated no one noticed this remaining line